basics.md

```diff
@@ -0,0 +1,108 @@
+Alright so you have installed the repository. Good job 🎉
+now let's began with writing some code-?
+
+**here's how to define a `variable`**
+Variable names can be arbitrarily long
+```
+
+```
+
+to **print** these, this is what you do
+```
+
+```
+
+you can't use variables with these names-
+```
+
+
+
+
+
+
+
+
+
+
+```
+
+**wanna add subtract multiply divide differentiate exponent or any mathematical funtion?**
```

Comment:
```suggestion
**wanna add subtract multiply differentiate exponent or any mathematical function?**
```
I was getting undefined when I ran a division on `   ` so I don't think it should be included in the operations.